docs/content/docs/getting_started/discover_heimdall.adoc

```diff
@@ -14,20 +14,20 @@ description: This page gives an overview about heimdall, the challenges it addre
 
 == What is heimdall?
 
-Heimdall is a cloud native identity aware proxy and access control decision service inspired by the Zero Trust idea. It brings together authentication and authorization systems and can be thought as an orchestrator for these in front of your services, allowing however completely retaining control even without the need for any type of maintenance in your own code.
+Heimdall is a cloud native identity aware proxy and access control decision service inspired by the Zero Trust idea. It brings together authentication and authorization systems and can be thought of as an orchestrator for these in front of your services, allowing you to completely retain control without the need for any type of maintenance in your own code.
 
-You can use it
+You can use it, in one of the following modes:
 
-* integrated into available proxies and API gateways, allowing implementation of Edge-level Authorization Architectures, or
-* stand alone as an authentication & authorization proxy
+* Integrated into existing proxies and API gateways, enabling the implementation of Edge-level Authorization Architectures
+* A standalone solution, serving as an authentication and authorization proxy.
 
-Both approaches can help you scaling and securing you services by transparently adding security capabilities and secure defaults related to authentication and authorization.
+Both approaches can help with scaling and securing your services. By transparently adding security capabilities and secure defaults, related to authentication and authorization.
```

Comment:
The added dot makes the new second sentence feel incomplete, at least from my perspective.